### ISSUE TITLE
swcarpentry/git-novice

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#72]
tracking issue for <https://github.com/swcarpentry/git-novice>

 - [x] fix links
   - [x] fix reference.md reference in index
   - [x] fix filter to allow github-create-repo-0X.png figures
 - [x] fix definition list anchors
 - [x] make table in episode 2 wider on the second column
 - [x] fix caption in https://github.com/swcarpentry/git-novice/blob/edf29f57814f6ab5c08251c3560178248297733c/_episodes/05-history.md?plain=1#L320 (the slashes are escaped)

```markdown

episodes/01-basics.md:25 [image missing alt-text]: fig/phd101212s.png
episodes/05-history.md:303 [image missing alt-text]: fig/git_staging.svg
episodes/07-github.md:39 [missing file]: [](fig/github-create-repo-01.png)
episodes/07-github.md:48 [missing file]: [](fig/github-create-repo-02.png)
episodes/07-github.md:53 [missing file]: [](fig/github-create-repo-03.png)
episodes/14-supplemental-rstudio.md:85 [image missing alt-text]: fig/RStudio_screenshot_navigateexisting.png
episodes/14-supplemental-rstudio.md:98 [image missing alt-text]: fig/RStudio_screenshot_editfiles.png
episodes/14-supplemental-rstudio.md:112 [image missing alt-text]: fig/RStudio_screenshot_review.png
episodes/14-supplemental-rstudio.md:134 [image missing alt-text]: fig/RStudio_screenshot_viewhistory.png
index.md:20 [missing file]: [version control](reference.md#version-control)
instructors/instructor-notes.md:39 [uninformative link text]: [this](https://github.com/rgaiacs/swc-shell-split-window)
```

**Comment by kekoziar** [comment]
pinging issue to get notifications.

**Comment by zkamvar** [comment]
I've updated the lesson and fixed any problems that I found. I believe it is ready for transition on Monday

**Comment by zkamvar** [comment]
fixed in c0ed3e58f6294818cf41749d2fe38c9e7a207607
